backend/request.go

```diff
@@ -86,6 +87,55 @@ func GetHost(r *http.Request) string {
 	return "https://" + r.Host
 }
 
+func stringMap(vs []string, f func(string) string) []string {
+	vsm := make([]string, len(vs))
+	for i, v := range vs {
+		vsm[i] = f(v)
+	}
+	return vsm
```

Comment:
I like the GO way of naming variables....not.

Reply:
(it's fine)

Reply:
I'm not even sure what's going on here, and I'm a short variable name aficionado … this is basically copy-and-pasted from https://gobyexample.com/collection-functions.